api/v1/openapi.yaml

```diff
@@ -664,6 +664,9 @@ definitions:
       policy:
         description: Policy information of endpoint
         "$ref": "#/definitions/EndpointPolicy"
+      policy-enabled:
```

Comment:
I tried doing this, but this results in the flag needing to reside within the policy.Consumable for the Endpoint, which is not created when the daemon receives an API call to create a new endpoint; the policy.Consumable object is created later in the lifetime of endpoint, when labels are added to a container: `daemon/endpoint.go:func (h *putEndpointIDLabels) Handle(params PutEndpointIDLabelsParams) middleware.Responder `
this function eventually calls `pkg/endpoint/policy.go:SetIdentity(owner Owner, id *policy.Identity)`, which then finally creates the consumable. 

I think tying the flag to the creation of the consumable is not desired behavior here.